packages/sdk/src/wallet/Ordit.ts

```diff
@@ -119,11 +114,18 @@ export class Ordit {
     return this.allAddresses
   }
 
-  setDefaultAddress(type: AddressFormats, index = 0) {
+  setDefaultAddress(type: AddressFormats, accountIndex = 0, addressIndex = 0) {
```

Comment:
saem for this params lets create an `opts?` param instead

Reply:
updated, made it a a bit different because there's default values for these two